modules/ROOT/edits/bind-api-configure-service-mesh-CLI.adoc

```diff
@@ -25,13 +23,13 @@ nto-payments   customers   anypoint-service-mesh-adapter   1234    customers
 
 == Methods for Binding an API
 
-You can bind an API to a service using Anypoint Platform or Anypoint Service Mesh, based on your preference:
+You can bind an API to a service using API ID or Anypoint Service Mesh API name, based on your preference:
```

Comment:
"using API ID" should be "using an API ID".

Reply:
Done! Thanks!